plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -35,18 +51,22 @@ test.describe('Customers API tests: CRUD', () => {
 			expect(response.status()).toEqual(200);
 			expect(responseJSON.is_paying_customer).toEqual(false);
 			expect(responseJSON.role).toEqual('administrator');
-			expect(responseJSON.username).toEqual('admin');
+			// this test was updated to allow for local test setup and JN
```

Comment:
Let's update this comment so it's more agnostic to platform:

```suggestion
			// this test was updated to allow for local test setup and other test sites.
```

Reply:
That's a good shout 👍